consensus/hotstuff/forks/forks.go

```diff
@@ -26,8 +26,8 @@ var ErrPrunedAncestry = errors.New("cannot resolve pruned ancestor")
 //	[b<-qc_b]  [b'<-qc_b']  [b*]
 type ancestryChain struct {
```

Comment:
Don't know if we would like to do this as part of this PR, but `model.CertifiedBlock` consists of two pointers, I understand we have legacy code, but I would be in favor of replacing `*model.CertifiedBlock` with `model.CertifiedBlock`.

Reply:
thanks. Totally agree. I was planning to work on that as part of my next PR, where I extend and clean up forks more broadly